diff --git a/node_modules/carbon-components-svelte/src/ComposedModal/ComposedModal.svelte b/node_modules/carbon-components-svelte/src/ComposedModal/ComposedModal.svelte
index bd7fa82..130d4a7 100644
--- a/node_modules/carbon-components-svelte/src/ComposedModal/ComposedModal.svelte
+++ b/node_modules/carbon-components-svelte/src/ComposedModal/ComposedModal.svelte
@@ -19,26 +19,20 @@
   export let preventCloseOnClickOutside = false;
 
   /** Specify a class for the inner modal */
-  export let containerClass = "";
+  export let containerClass = '';
 
   /**
    * Specify a selector to be focused when opening the modal
    * @type {null | string}
    */
-  export let selectorPrimaryFocus = "[data-modal-primary-focus]";
+  export let selectorPrimaryFocus = '[data-modal-primary-focus]';
 
   /** Obtain a reference to the top-level HTML element */
   export let ref = null;
 
-  import {
-    createEventDispatcher,
-    tick,
-    setContext,
-    onMount,
-    afterUpdate,
-  } from "svelte";
-  import { writable } from "svelte/store";
-  import { trackModal } from "../Modal/modalStore";
+  import { createEventDispatcher, tick, setContext, onMount, afterUpdate } from 'svelte';
+  import { writable } from 'svelte/store';
+  import { trackModal } from '../Modal/modalStore';
 
   const dispatch = createEventDispatcher();
   const label = writable(undefined);
@@ -47,26 +41,25 @@
   let innerModal = null;
   let didClickInnerModal = false;
 
-  setContext("ComposedModal", {
+  setContext('ComposedModal', {
     closeModal: () => {
       open = false;
     },
     submit: () => {
-      dispatch("submit");
-      dispatch("click:button--primary");
+      dispatch('submit');
+      dispatch('click:button--primary');
     },
-    declareRef: (ref) => {
+    declareRef: ref => {
       buttonRef = ref;
     },
-    updateLabel: (value) => {
+    updateLabel: value => {
       label.set(value);
-    },
+    }
   });
 
   function focus(element) {
     if (selectorPrimaryFocus == null) return;
-    const node =
-      (element || innerModal)?.querySelector(selectorPrimaryFocus) || buttonRef;
+    const node = (element || innerModal)?.querySelector(selectorPrimaryFocus) || buttonRef;
     if (node != null) node.focus();
   }
 
@@ -87,11 +80,11 @@
     if (opened) {
       if (!open) {
         opened = false;
-        dispatch("close");
+        dispatch('close');
       }
     } else if (open) {
       opened = true;
-      dispatch("open");
+      dispatch('open');
     }
   });
 </script>
@@ -105,11 +98,11 @@
   class:bx--modal--danger={danger}
   {...$$restProps}
   on:keydown
-  on:keydown={(e) => {
+  on:keydown={e => {
     if (open) {
-      if (e.key === "Escape") {
+      if (e.key === 'Escape') {
         open = false;
-      } else if (e.key === "Tab") {
+      } else if (e.key === 'Tab') {
         // taken from github.com/carbon-design-system/carbon/packages/react/src/internal/keyboard/navigation.js
         const selectorTabbable = `
   a[href], area[href], input:not([disabled]):not([tabindex='-1']),
@@ -140,8 +133,8 @@
   on:mouseenter
   on:mouseleave
   on:transitionend={({ propertyName, currentTarget }) => {
-    if (propertyName === "transform") {
-      dispatch("transitionend", { open });
+    if (propertyName === 'transform') {
+      dispatch('transitionend', { open });
     }
 
     if (didOpen) {
@@ -150,17 +143,16 @@
     }
   }}
 >
-  <!-- svelte-ignore a11y-click-events-have-key-events -->
-  <!-- svelte-ignore a11y-no-noninteractive-element-interactions -->
+  <!-- svelte-ignore a11y-click-events-have-key-events, a11y-no-noninteractive-element-interactions, a11y_interactive_supports_focus -->
   <div
     bind:this={innerModal}
     role="dialog"
     aria-modal="true"
-    aria-label={$$props["aria-label"] || $label || undefined}
+    aria-label={$$props['aria-label'] || $label || undefined}
     class:bx--modal-container={true}
-    class:bx--modal-container--xs={size === "xs"}
-    class:bx--modal-container--sm={size === "sm"}
-    class:bx--modal-container--lg={size === "lg"}
+    class:bx--modal-container--xs={size === 'xs'}
+    class:bx--modal-container--sm={size === 'sm'}
+    class:bx--modal-container--lg={size === 'lg'}
     class={containerClass}
     on:click={() => {
       didClickInnerModal = true;
